packages/graphql/src/services/notifications.ts

```diff
@@ -49,3 +52,121 @@ export async function subscribeToPush(
     },
   })
 }
+
+interface UnsubscribeToPushArgs {
+  courseId: string
+  endpoint: string
+}
+
+export async function unsubscribeFromPush( 
+  { courseId, endpoint }: UnsubscribeToPushArgs,
+  ctx: ContextWithUser
+) {
+  try {
+    await ctx.prisma.pushSubscription.delete({
+      where: {
+        participantId_courseId_endpoint: {
+          participantId: ctx.user.sub,
+          courseId,
+          endpoint
+        }
+      }
+    })
+    return true
+  } catch(error) {
+    console.error("An error occured while trying to unsubscribe from push notifications: ", error)
+    return false
+  }
+}
+
+export async function sendPushNotifications( ctx: Context) {
+  webpush.setVapidDetails(
```

Comment:
check beforehand if VAPID\_PUBLIC\_KEY and VAPID\_PRIVATE\_KEY are set. if not, return early. also, the support email used here should be customizable using an env variable (like we did with the cron token).